src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const list = document.getElementById('thumbs');
+const bigImage = document.getElementById('largeImg');
+
+list.addEventListener('click', (e) => {
+  e.preventDefault();
```

Comment:
The `e.preventDefault();` call is used to prevent the default action of the event. However, it should be placed inside the `if (item)` block to ensure it only prevents the default action when a valid list item is clicked. Otherwise, it might prevent default actions for other clicks within the `list` element that are not on a `.list-item`.